mscore/musescore.cpp

```diff
@@ -964,92 +951,156 @@ MuseScore::MuseScore()
+          menuAddPitch->addAction(a);
+      }
       for (int i = 0; i < 7; ++i) {
-            char buffer[8];
-            sprintf(buffer, "chord-%c", "cdefgab"[i]);
-            a = getAction(buffer);
-            menuAddPitch->addAction(a);
-            }
-
-      QMenu* menuAddInterval = new QMenu(tr("Add &Interval"));
+          char buffer[8];
+          sprintf(buffer, "chord-%c", "cdefgab"[i]);
+          a = getAction(buffer);
+          menuAddPitch->addAction(a);
+      }
+      menuCreate->addMenu(menuAddPitch);
+          
+      QMenu* menuAddInterval = new QMenu(tr("&Intervals"));
       for (int i = 1; i < 10; ++i) {
-            char buffer[16];
-            sprintf(buffer, "interval%d", i);
-            a = getAction(buffer);
-            menuAddInterval->addAction(a);
-            }
+          char buffer[16];
+          sprintf(buffer, "interval%d", i);
+          a = getAction(buffer);
+          menuAddInterval->addAction(a);
+      }
       menuAddInterval->addSeparator();
       for (int i = 2; i < 10; ++i) {
-            char buffer[16];
-            sprintf(buffer, "interval-%d", i);
-            a = getAction(buffer);
-            menuAddInterval->addAction(a);
-            }
-      menuNotes->addMenu(menuAddInterval);
+          char buffer[16];
+          sprintf(buffer, "interval-%d", i);
+          a = getAction(buffer);
+          menuAddInterval->addAction(a);
+      }
+      menuCreate->addMenu(menuAddInterval);
 
-      QMenu* menuNtole = new QMenu(tr("T&uplets"));
+      QMenu* menuTuplet = new QMenu(tr("T&uplets"));
       for (auto i : { "duplet", "triplet", "quadruplet", "quintuplet", "sextuplet",
-         "septuplet", "octuplet", "nonuplet", "tuplet-dialog" })
-            menuNtole->addAction(getAction(i));
-      menuNotes->addMenu(menuNtole);
-
-      menuNotes->addSeparator();
-      menuNotes->addAction(getAction("transpose"));
-      a = getAction("concert-pitch");
-      a->setCheckable(true);
-      menuNotes->addAction(a);
+          "septuplet", "octuplet", "nonuplet", "tuplet-dialog" })
+          menuTuplet->addAction(getAction(i));
+      menuCreate->addMenu(menuTuplet);
+          
+      menuCreate->addSeparator();
+          
+      QMenu* menuAddMeasures = new QMenu(tr("&Measures"));
+      menuAddMeasures->addAction(getAction("insert-measure"));
+      menuAddMeasures->addAction(getAction("insert-measures"));
+      menuAddMeasures->addSeparator();
+      menuAddMeasures->addAction(getAction("append-measure"));
+      menuAddMeasures->addAction(getAction("append-measures"));
+      menuCreate->addMenu(menuAddMeasures);
+          
+      QMenu* menuAddFrames = new QMenu(tr("&Frames"));
+      menuAddFrames->addAction(getAction("insert-hbox"));
+      menuAddFrames->addAction(getAction("insert-vbox"));
+      menuAddFrames->addAction(getAction("insert-textframe"));
+      if (enableExperimental)
+          menuAddFrames->addAction(getAction("insert-fretframe"));
```

Comment:
No, more. Second line should be indented with 6 spaces compared to the first one. Check how it's done elsewhere with a "if"

```
if (enableExperimental)
      menuAddFrames->addAction(getAction("insert-fretframe"));
```